child_compassion/wizards/global_child_search.py

```diff
@@ -54,7 +54,28 @@ class GlobalChildSearch(models.TransientModel):
 
     # Pagination
     skip = fields.Integer(size=4)
-    take = fields.Integer(default=80, size=4)
+
+    def compute_number_reserve_children(self):
```

Comment:
Fields method should be declared after the fields. To be able to call a method in the default param you can use:
```
default=lambda self: self.compute_number_reserve_children()
```
Please rename compute methods to `default_number_reserve_children()` to avoid confusing them with computed fields methods.